controllers/scopetemplate_controller_test.go

```diff
@@ -235,7 +235,7 @@ var _ = Describe("ScopeTemplate", func() {
 
 						verifyRoleBindings(existingRB, scopeInstance, scopeTemplate)
 
-						roleBindingList = listRoleBinding(namespace.GetName(), 0, labels)
+						_ = listRoleBinding(namespace.GetName(), 0, labels)
```

Comment:
I know the linter was complaining about an unused variable. But why are we removing it instead of doing some sort of verification on it.

Also, why do we even return a list of RoleBindings if that method takes in an expected number. 

Seems to me `listRoleBinding` should return an error if it doesn't match the expected number.

Reply:
Yes, this verification is when one of the namespaces is removed and then ScopeInstance is updated then it should delete Role Bindings from that namespace. I updated the code.

Reply:
We are getting the entire object for role binding and then the `verifyRoleBindings` function is verifying all its values. That's why we are returning it in `listRoleBinding`.